lib/fluent/plugin/out_amqp.rb

```diff
@@ -53,6 +53,7 @@ class AMQPOutput < Output
 
     config_section :buffer do
       config_set_default :@type, DEFAULT_BUFFER_TYPE
+      config_set_default :chunk_keys, ['tag']
```

Comment:
This `config_section :buffer` block is equivalent to the following buffer directive:

```
<buffer tag>
  @type memory
</buffer>
```

This block ensures `tag` in chunk_keys.
When tag is specified in chunk_keys, Fluentd splits chunk every tag.